.circleci/config.yml

```diff
@@ -3,6 +3,18 @@ version: 2.1
 # ⚠️ If you add, rename or delete a job here, please also update .mergify.yml! ⚠️
 
 commands:
+  # Our policy for updating rust versions is at https://github.com/mozilla/application-services/tree/main/docs/rust-versions.md
+  # See also rust-toolchain in the root of this repo, which is used to specify our official target version.
+  setup-rust-target-version:
+    steps:
+      # So long as this is executed after the checkout it will use the version specified in rust-toolchain.yaml
+      - run: rustup update
+  # Our minimum supported rust version is specified here.
+  setup-rust-min-version:
+    steps:
+      # Technically should be on 1.47 here, but that boat sailed - 1.51 is min.
```

Comment:
Since our policy is "must be buildable with the min version" and the problem is in a test crate, an alternative here could be to use `1.47` and only have CI execute a `cargo build` or equivalent, rather than actually running the tests. I think this would skip trying to build `ctest2` because it's not in the workspace default members.